lib/src/visitor/perform.dart

```diff
@@ -555,11 +565,29 @@ class _PerformVisitor
     });
   }
 
+  Uri _resolvePackageUri(DynamicImport import) {
+    var packageUri = import.url;
+    if (packageUri.scheme == 'package') {
+      if (packageResolver == null)
```

Comment:
Nit: include `{}` even for single-statement `if`s (see [the style guide](https://www.dartlang.org/guides/language/effective-dart/style#do-use-curly-braces-for-all-flow-control-structures)).